src/metrics/policy/storage_policy.go

```diff
@@ -270,3 +270,28 @@ func (sp ByResolutionAscRetentionDesc) Less(i, j int) bool {
 	}
 	return sp[i].Resolution().Precision < sp[j].Resolution().Precision
 }
+
+// ByRetentionAscResolutionAsc implements the sort.Sort interface that enables sorting
+// storage policies by retention in ascending order and then by resolution in ascending
+// order.
+type ByRetentionAscResolutionAsc StoragePolicies
+
+func (sp ByRetentionAscResolutionAsc) Len() int      { return len(sp) }
+func (sp ByRetentionAscResolutionAsc) Swap(i, j int) { sp[i], sp[j] = sp[j], sp[i] }
+func (sp ByRetentionAscResolutionAsc) Less(i, j int) bool {
+	rt1, rt2 := sp[i].Retention(), sp[j].Retention()
+	if rt1 < rt2 {
```

Comment:
Nit: can simplify this to 

if rt1 != rt2 {
  return rt1 < rt2
}
// rw1, rw2 ...

Reply:
Fair, will update the other sorter to be the same